dora/core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -136,6 +136,13 @@ public void before() throws Exception {
     // set a large value of PropertyKey.MASTER_LOST_WORKER_DELETION_TIMEOUT_MS
     // to prevent worker to be deleted after it is lost
     Configuration.set(PropertyKey.MASTER_LOST_WORKER_DELETION_TIMEOUT_MS, Integer.MAX_VALUE);
+    Configuration.set(PropertyKey.MASTER_TIERED_STORE_GLOBAL_LEVELS, 3);
```

Comment:
wanna mark this one `@Ignore` so you don't have to make the changes? I'm removing this test very soon.

Reply:
I think it doesn't matter if we decide to remove it eventually